### ISSUE TITLE
Develop backend updates to Settings Page (Facility Addres, Facility Phone, Swab Type)

### ISSUE THREAD

**Issue by willusds** [CDCgov/prime-central#153]
Blocked by #152

Engineering work to add fields

- [ ]  Facility Address
- [ ]  Facility Phone
- [ ]  Swab Type (similar to devices)
- [ ]  Add swab to test queue cards too

See #106 to see updates about Swab Type

**Comment by aliciabeckett-gov** [comment]
Moving swab type to a new ticket
